### PR TITLE
test 3.6 and 3.7 too

### LINKED ISSUE #132: Add support for Python 3.6 and Python 3.7
Add 3.6 and 3.7 to Travis and make them pass.

### PR DESCRIPTION
**Pull request by JelleZijlstra**
Fixes #132

### CODE REVIEW

**Comment by jettify** [comment]
Probably good idea to use `allow_failures` for 3.7 before release.

File: .travis.yml [hunk 1]
```diff
@@ -3,13 +3,15 @@ language: python
 python:
   - "3.4"
   - "3.5"
+  - "3.6"
+  - "3.7-dev"
```
Comment (webknjaz) [inline .travis.yml]: Please change to what I recommended in #144 (stable 3.7.0)

Comment (JelleZijlstra) [reply .travis.yml]: I'll just abandon this PR for now and try again after we get 3.7 Travis running on master.

Comment (webknjaz) [reply .travis.yml]: Adding it with allowed failures allows everyone see the status and pick up some tests for fixing. Thus it would be beneficial :)

**Pull request closed without merging**